__tests__/app.test.js

```diff
@@ -0,0 +1,37 @@
+process.env.NODE_ENV = "test";
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const db = require("../db/connection");
+const app = require("../app.js");
+const request = require("supertest");
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("get api topics", () => {
+  test("200 : should respond with an array of topic objects with the properties slug and description", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((data) => {
+        data.body.forEach((topic) => {
```

Comment:
what's to say you're responding with an empty array here? How many topics should come back?

Reply:
Added a .toHaveLength to check how many results are returned